tests/objects/pickBy.test.js

```diff
@@ -0,0 +1,32 @@
+const pickBy = require('../../src/objects/pickBy');
+
+describe('pickBy', () => {
+  test("Creates an object composed of the object properties predicate returns truthy for.", () => {
+    var object = { 'a': 1, 'b': '2', 'c': 3 };
+    var objectSec = { 'a': 'aaa', 'b': '2', 'c': 3 , 'd': '44'};
```

Comment:
also, please check the variables declarations, please use 'const' where it's possible and 'let' where using 'const' is impossible